cybersixgill_actionable_alerts/assets/dashboards/cybersixgill_actionable_alerts_overview.json

```diff
@@ -0,0 +1,132 @@
+{
+    "title":"fshaik's Dashboard Thu, Sep 1, 4:41:47 pm",
```

Comment:
Please change the dashboard name to reflect the integration name. Ex: Cybersixgill